File: app/src/main/java/com/rikkimikki/torrentino/domain/pojo/search/Top.java
package com.rikkimikki.torrentino.domain.pojo.search;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Top {
    @SerializedName("topResult")
    @Expose
    private TopResult topResult;
    @SerializedName("movies")
    @Expose
    private List<Movie> movies = null;

    public TopResult getTopResult() {
        return topResult;
    }

    public void setTopResult(TopResult topResult) {
        this.topResult = topResult;
    }

    public List<Movie> getMovies() {
        return movies;
    }

    public void setMovies(List<Movie> movies) {
        this.movies = movies;
    }
}

File: app/src/main/java/com/rikkimikki/torrentino/domain/pojo/search/Top.kt
package com.rikkimikki.torrentino.domain.pojo.search

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Top (
    @SerializedName("topResult")
    @Expose
    val topResult: TopResult,

    @SerializedName("movies")
    @Expose
    val movies: List<Movie>
)